gradle.properties

```diff
@@ -1,2 +1,25 @@
 release.scope=patch
 release.version=3.0.0-SNAPSHOT
+VERSION_NAME=3.0.0-SNAPSHOT
```

Comment:
Can we unify this with the variable above?

Reply:
Done.